dap_prinz_green_jobs/pipeline/green_measures/industries/industries_measures.py

```diff
@@ -18,8 +18,6 @@
     'INDUSTRY PROP HOURS GREEN TASKS': 9.700000000000001,
     'INDUSTRY PROP WORKERS GREEN TASKS': 43.5,
     'INDUSTRY PROP WORKERS 20PERC GREEN TASKS': 23.599999999999998}]]
```

Comment:
I think this example isn't updated with the latest output dict structure? And again, the example here gives me no SIC found